### ISSUE TITLE
Not able to upgrade or install Angular CLI on macOS

### ISSUE THREAD

**Issue by jms-1005** [angular/angular-cli#24811]
### Which @angular/* package(s) are the source of the bug?

core

### Is this a regression?

Yes

### Description

I tried upgrading but it failed. So I uninstalled it globally and then tried reinstalling and get this error.

sudo npm install -g @angular/cli
Password:
npm ERR! code E404
npm ERR! 404 Not Found - GET https://skimdb.npmjs.com/registry/@angular/cli/-/cli-15.2.1.tgz - not_found
npm ERR! 404
npm ERR! 404 '@angular/cli@https://skimdb.npmjs.com/registry/@angular/cli/-/cli-15.2.1.tgz' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
[2023-03-04T15_48_32_998Z-debug-0.txt](https://github.com/angular/angular/files/10889093/2023-03-04T15_48_32_998Z-debug-0.txt)

### Please provide the exception or error you saw

```true
Error attached
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
14.0.2
```

**Comment by JoostK** [comment]
The primary registry (`https://registry.npmjs.com/@angular/cli/-/cli-15.2.1.tgz`) does provide it, so this appears to be some sort of sync delay with the `skimdb` registry mirror. This is not caused by Angular, so there's nothing we can do to resolve this issue.

**Comment by jms-1005** [comment]
But, can you please suggest a solution to this problem?

**Comment by alan-agius4** [comment]
This seems to be a problem with npm skim db replica, this can be either caused by misconfiguration or a bug on their side. 

Quickest workaround would be to use the NPM public registry directly.

**Comment by jms-1005** [comment]
Can you please proive a link to resources that can help me do that? Or, can you give me a solution for that? I tried looking around without any luck! Thanks!

**Comment by jms-1005** [comment]
I like Angular and have been using it for the past 5-6 years but this incident is an eye-opener on how the support for Angular is not optimal and brings many questions about whether this platform can be trusted. I posted the issue here and on Stackoverflow and there has been little to no help at all. No wonder react is taking over. 

BTW, I used brew to install angular CLI and it is working fine. The issue with NPM is still unresolved. For those who land on this page with a similar error, use this instead - https://formulae.brew.sh/formula/angular-cli
